.travis.yml

```diff
@@ -15,4 +15,4 @@ install:
 script:
   - vendor/bin/phpunit
   - bin/psalm
-  - vendor/bin/phpcs
+  - vendor/bin/php-cs-fixer fix --verbose --allow-risky=yes --dry-run
```

Comment:
Here, you map which files you want to consider under CS restrictions. it may be 95% of files in your project.
No need to validate all of them on Travis, if only one file was modified in PR.